inframon/tools/tool.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing, please email
+# opensource@seagate.com or cortx-questions@seagate.com.
+
+
+""" A Base class for implementing a Tool which
+    interacts with the hardware or service to be monitored.
+"""
+
+
+from abc import ABC, abstractmethod
+
+import json
+import math
+from typing import Any
+
+
+class ResponseClassMarker(object):
+
+    @staticmethod
+    def format_dict(d) -> Any:
+        p_d = {}
```

Comment:
can p_d, k, v, d renamed to understandable variable names

Reply:
This works on any dictionary and these single char variables make more sense here. There are no specific key and value field.